sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -376,7 +385,9 @@ private void setArchitectures(final @NotNull Device device) {
       device.setScreenDpi(displayMetrics.densityDpi);
     }
 
+    // likely to be applied only if applyScopeData
```

Comment:
the chances that the user opens the app again right away is high, so this won't change much